pyproject.toml

```diff
@@ -58,6 +58,9 @@ sotastream = "sotastream.cli:main"
 requires = ["setuptools", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[tool.setuptools.dynamic]
+version = {file = "sotastream/version.txt"}
+
 [tool.setuptools.packages.find]
```

Comment:
Okay, this is good. Any chance that this dynamic feature can parse `__version__ = "1.0.1"` directly from the package `__init__.py` file? That would maybe be more natural, would simplify the code in the package init file, and would save us creating `version.txt`.

Reply:
`version = {attr = "sotastream.__version__"}` is supposed to do it. 

Previously, it didnt work on clean environment where requirements were not yet installed.

I tried again now and it seems to work. Maybe the newer pip/setuptools addressed the probjem.. It installs requirements first and then accesses `sotastream.__version__`. 

Please test it and let me know.

Reply:
Works locally, LGTM